openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/TypeRepository.java

```diff
@@ -187,6 +190,27 @@ private void validateProperty(CustomProperty customProperty) {
           throw new IllegalArgumentException("Enum Custom Property Type must have EnumConfig.");
         }
       }
+      case "metaEnum" -> {
+        CustomPropertyConfig config = customProperty.getCustomPropertyConfig();
+        if (config != null) {
+          MetaEnumConfig metaEnumConfig =
+              JsonUtils.convertValue(config.getConfig(), MetaEnumConfig.class);
+          if (metaEnumConfig == null
```

Comment:
can we create validate method and handle these checks there?

Reply:
added new validate method